cdk/resources/ConfigureDevice.ts

```diff
@@ -23,16 +23,48 @@ export class ConfigureDevice extends Construct {
 			lambdaSources,
 			layers,
 			websocketAPI,
+			nRFCloudAccounts,
 		}: {
 			websocketAPI: WebsocketAPI
 			lambdaSources: {
 				configureDevice: PackedLambda
 			}
 			layers: Lambda.ILayerVersion[]
+			nRFCloudAccounts: string[]
 		},
 	) {
 		super(parent, 'configureDevice')
 
+		const policies = nRFCloudAccounts
+			.map((account) => [
+				new IAM.PolicyStatement({
+					actions: ['ssm:GetParameter'],
+					resources: [
+						`arn:aws:ssm:${Stack.of(this).region}:${
+							Stack.of(this).account
+						}:parameter/${Stack.of(this).stackName}/thirdParty/${account}/*`,
```

Comment:
Simplify this by using wildcard.
```suggestion
						}:parameter/${Stack.of(this).stackName}/thirdParty/*`,
```